packages/documentation-site/patternfly-docs/content/design-guidelines/components/alert/alert.md

```diff
@@ -70,8 +69,46 @@ The overflow link should be configured to navigate to a place where the user can
 
 See our [toast alert guidelines](/components/alert/design-guidelines/#using-toast-alerts) for more information about using toast alerts.
 
+## Placement
+Toast alerts should be placed in an alert group in the upper-right corner of a page. Use 8px of padding between each alert in an alert group. Alert group alignment on a page will depend on your use case.
+
+### Toast alerts
+Toast alerts communicate information about an update, confirmation, or other activity. A toast alert overlays content in the upper-right corner of the page and disappears when it times out or when the user dismisses it.
+ 
+![toast notification placement](./img/toast-notification.png)
+ 
+
+#### Using toast alerts
+Use toast alerts to communicate an update, confirmation, or other activity to the user without blocking their workflow. For example, you can use a toast alert to inform the user that their message was sent. You can include links in a toast alert for a user to take action on the event that surfaced the alert, but the links should only be a shortcut for the user. A toast alert should never be used as the only means for taking actions on an alert event.
+ 
+Toast alerts stack in the upper-right corner of a page. To properly position and stack a set of alerts, use an [alert group](#alert-groups) component. When alerts stack, the most recent alert appears at the top, and others move down as more alerts come in. When alerts are deleted, the remaining alerts move up.
+ 
+![alert group](./img/toast-notification-multiple.png)
+ 
+Toast alerts should always have a close button so that the user can dismiss them. If you include an automatic timeout for your toast alert, 8 seconds is recommended. Toast alerts should not disappear while the user is hovering over them.
+ 
+<img src="./img/Toast-alerts-desktop.png" alt="toast alerts in a page" />
+ 
+Toast alerts are commonly used in the following situations:
+- Confirming that a message sent
+- Confirming that an update was made
+- Informing the user that their action was completed successfully
+- Informing the user that their action was completed with errors
+
+**For desktop views**, right align toast alert groups over your page content. Each alert should stack on top of one another with 8px of padding in between.
+
+<img src="./img/alert-group-desktop.png" alt="An example of a right-aligned toast alert group over a desktop view." />
+
+**For mobile views**, center toast alert groups over your page content.
+
+<img src="./img/alert-group-mobile.png" alt="An example of a centered toast alert group over a mobile view." width="375" />
 
-## Variations
+<img src="./img/alert-group-mobile.png" alt="An example of a centered toast alert group over a mobile view." width="375" />
+
+See our [toast alert guidelines](/components/alert/design-guidelines/#using-toast-alerts) for more information about using toast alerts.
+
+
+### Inline alerts
```

Comment:
Would delete this header!

Reply:
So the ###bordered inline alerts become the header for rest of the inline alerts content, right?